qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/plot_widget/plotting_canvas/plotting_canvas_model.py

```diff
@@ -4,21 +4,29 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
-from typing import NamedTuple, List
+from typing import List
 from mantidqtinterfaces.Muon.GUI.Common.utilities.algorithm_utils import run_convert_to_points, run_convert_to_histogram
 from mantidqtinterfaces.Muon.GUI.Common.ADSHandler.ADS_calls import retrieve_ws
 
 
 FIT_FUNCTION_GUESS_LABEL = "Fit function guess"
 
 
-class WorkspacePlotInformation(NamedTuple):
-    workspace_name: str
-    index: int
-    axis: int
-    normalised: bool
-    errors: bool
-    label: str
+class WorkspacePlotInformation(object):
+
+    def __init__(self,
```

Comment:
Just for the future to save typing, we have dataclasses in Python which remove all the boilerplate from classes like [this](https://docs.python.org/3/library/dataclasses.html).

This doesn't have to be done for this PR, just something to save you writing as much boilerplate next time :wink: 

To use them in Mantid we simply use:
`from mantid.py36compat import dataclass` instead